MementoMori/MementoMori.Server/CardFile/FileWriter.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.IO;
+namespace MementoMori.Server
+{
+    public class FileWriter
+    {
+        private readonly string _directoryPath;
+
+        // Constructor to initialize the directory where files will be stored
+        public FileWriter()
+        {
+            // Get the current directory of the MementoMori.Server project
+            string serverDirectory = Directory.GetCurrentDirectory();
+
+            // Set the path to "CardFile" folde
+            _directoryPath = Path.Combine(serverDirectory, "CardFile");
+
+        }
+
+        // Method to create a file using tags and text
+        public void CreateFile(string tags, string text, string deckId)
```

Comment:
```suggestion
        public void CreateFile(Guid deckId, string text, string tags)
```

Use Guid's for ids for consistency.
Imo this ordering is more understandable, unique identifiers and important data should come first.

Reply:
Wont fix